extensions/wikibase/src/org/openrefine/wikibase/editing/MediaFileUtils.java

```diff
@@ -49,6 +49,8 @@ public class MediaFileUtils {
     protected String csrfToken = null;
     protected int maxLagWaitTime = 5000; // ms
     protected String filePrefix = "File:"; // configurable?
+    protected double backoffFactor = 1.5;
```

Comment:
What's the maximum total wait time using these values? I don't know how to do the math quickly, but staring at 5 seconds and increasing by half up to 14 time sounds like you'd end up with a pretty big number.

Reply:
Max wait time 1598 seconds

Reply:
I wouldn't mark this conversation as resolved, as there could still be the decision to change some of those parameters to get a shorter maximum wait.